packages/components/progress/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const PROGRESS_NAME = 'Progress'
```

Comment:
why here component name

Reply:
The constant is reused in some methods in utils so let's just create a file that hold  the constants

Reply:
hmm okay `const INDICATOR_NAME = 'OkuProgressIndicator' but why dont add ?